elastalert/util.py

```diff
@@ -596,19 +643,27 @@ def get_version_from_cluster_info(client):
         try:
             esinfo = client.info()['version']
             esversion = esinfo['number']
-            if esinfo.get('distribution') == "opensearch":
+            distribution = esinfo.get('distribution')
+            if not is_not_empty(distribution):
```

Comment:
Possibly simplify `if not is_not_empty` logic (see earlier comment)

Reply:
Done